charts/ckan/values.yaml

```diff
@@ -83,7 +83,7 @@ ckan:
       create: true
       name: ""
     extraVolumeMounts: []
-    denyCrawlers: false
+    denyCrawlers: true
```

Comment:
Do we also need to change this conditional to allow search engine bots to index pages in production? 
https://github.com/alphagov/govuk-dgu-charts/blob/main/charts/ckan/templates/ckan/nginx-configmap.yaml#L93

```
        {{- if ( or (ne .Values.environment "production") (eq .Values.ckan.nginx.denyCrawlers true ) ) }}
        add_header X-Robots-Tag "noindex";
        {{- end }}
```